blazemeter-cloud/README.md

```diff
@@ -0,0 +1,71 @@
+# BlazeMeter Cloud Trial Image
+
+The [BlazeMeter Cloud](https://www.blazemeter.com/) trial job enables you to use BlazeMeter performance tests in your Optimize Pro experiments. It leverages a custom container image, based on the the official BlazeMeter Taurus([https://](https://gettaurus.org/) [image](https://hub.docker.com/r/blazemeter/taurus/).
```

Comment:
The link looks off. Maybe like this?

```suggestion
The [BlazeMeter Cloud](https://www.blazemeter.com/) trial job enables you to use BlazeMeter performance tests in your Optimize Pro experiments. It leverages a custom container image, based on the the official BlazeMeter ([Taurus](https://gettaurus.org/) [image](https://hub.docker.com/r/blazemeter/taurus/).
```

Reply:
Fixed in https://github.com/thestormforge/optimize-trials/pull/17/commits/9d1da3074b8c5d105ddd8a948898ae079c522f47